### PR TITLE
Sage-485: skip upgrade & clean-up apt steps

### PR DESCRIPTION
**Pull request by jswantek**

### CODE REVIEW

File: blade-image-build/ROOTFS/opt/sage/blade/image/base_packages [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh ansible git htop iotop iftop bwm-ng screen nmap sshfs autossh network-manager apt-transport-https ca-certificates curl gnupg-agent software-properties-common
```
Comment (ozorob2) [inline blade-image-build/ROOTFS/opt/sage/blade/image/base_packages:1]: This file should be referenced in the pressed file in the 'late_command' to copy it from the dockerfs to the bladefs
*see last line in pressed file, where we copy other files it's just a cp you just need to make sure the path is what is right, and we have found no problems with putting files on /root but putting them in specific locations like /etc/ has caused files to be truncated sometimes.

Comment (jswantek) [reply blade-image-build/ROOTFS/opt/sage/blade/image/base_packages:1]: so basically add this `cp /cdrom/wscripts/base_packages /target/root/base_packages` ??

Comment (jswantek) [reply blade-image-build/ROOTFS/opt/sage/blade/image/base_packages:1]: Okay i added it. thanks!

go ahead and resolve this conversation if you think I did it right.

Comment (ozorob2) [reply blade-image-build/ROOTFS/opt/sage/blade/image/base_packages:1]: Looks good I'll resolve thanks!

**Review by ozorob2 — APPROVED** [review]
Tested, and confirmed to work. Good to go

**Pull request merged**